translate.py

```diff
@@ -0,0 +1,249 @@
+"""
+Utility to download and generate timezone translations for the Wagtail admin
+from the Unicode Consortium CLDR. This only needs to be run if
+WAGTAILADMIN_PROVIDED_LANGUAGES changes or if timezones in pytz change.
+
+This script will output ``.po`` files with mostly automated translations.
+Language experts should review these and make necessary adjustments.
+"""
+
+# Import from standard library.
+import datetime
+import json
+import os
+import urllib
+
+# Setup django so we can import things from Wagtail.
+import django  # noqa
+
+os.environ.setdefault("DJANGO_SETTINGS_MODULE", "wagtail.test.settings")
+django.setup()
+
+
+import polib  # noqa
+import pytz  # noqa
```

Comment:
We've switched from `pytz` to `zoneinfo.available_timezones` in #12155, so this will need to be updated